_topics/en/compact-block-filters.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Compact block filters
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BIP157
+  - BIP158
+  - Neutrino protocol
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+  - Lightweight Client Support
+  - P2P Network Protocol
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Compact block filters** are a condensed representation of the
+  contents of a block that allow wallets to determine whether the block
+  contains any transactions affecting the wallet.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  A full node with a copy of a block and the data necessary to validate
+  that block uses [BIP158][] to create a Golomb-Rice Coded Set (GCS) of
+  data from that block (such as the prevout script from each input spent
+  in the block and the scriptPubKey for each output created in the
+  block).  The GCS (called a filter) is then distributed to wallets
+  (such as via the P2P protocol as described in [BIP157][]), allowing
+  them to search for any matches to each wallet's scripts.  If a match
+  is found, the wallet can then download the corresponding block to
+  access any relevant transactions.
+
+    The GCS mechanism guarantees that a properly-coded wallet will find
```

Comment:
remove 'properly-coded' here. It's confusing, especially in the context of something called 'Golomb-Rice Coding'